test/controllers/ReturnStatusControllerSpec.scala

```diff
@@ -73,4 +74,55 @@ class ReturnStatusControllerSpec
     }
 
   }
+
+  ".getCurrentReturns(commencementDate)" - {
+    val stubClock: Clock = Clock.fixed(LocalDate.of(2022, 10, 1).atStartOfDay(ZoneId.systemDefault).toInstant, ZoneId.systemDefault)
+    val period = Period(2021, Q2)
+    val period0 = Period(2021, Q3)
+    val period1 = Period(2022, Q1)
+    val period2 = Period(2022, Q2)
+    val period3 = Period(2022, Q3)
+    val periods = Seq(period, period0, period1, period2, period3)
+    val commencementDate = LocalDate.of(2021, 1, 1)
+    val returns = Returns(
+      None,
+      Some(Return.fromPeriod(period3)),
+      Seq(
+        Return.fromPeriod(period0),
+        Return.fromPeriod(period1),
+        Return.fromPeriod(period2)
+      )
+    )
+
+    lazy val request = FakeRequest(GET, routes.ReturnStatusController.getCurrentReturns(commencementDate).url)
+
+    "must respond with OK and a sequence of requests" in {
```

Comment:
Can we add more tests for this, specifically around different scenarios for for different outcomes of what `getStatuses` would produce. Alternatively, unit test that individually

Reply:
Don't we also usually have a standard set tests for unauthed responses?

Reply:
we added more specs now